packages/dm-core-plugins/src/form/fields/StringField.test.tsx

```diff
@@ -10,6 +16,30 @@ describe('StringField', () => {
   })
 
   describe('TextWidget', () => {
+    afterEach(() => {
+      cleanup()
+      jest.clearAllMocks()
+    })
+    it('should render a single string field', async () => {
+      mockBlueprintGet([
+        {
+          name: 'SingleField',
+          type: 'system/SIMOS/Blueprint',
+          attributes: [
+            {
+              name: 'foo',
+              type: 'system/SIMOS/BlueprintAttribute',
+              attributeType: 'string',
+            },
+          ],
+        },
+      ])
+      const { container } = render(<Form type="SingleField" />)
+      await waitFor(() => {
+        expect(container.querySelectorAll(` input[type=text]`).length).toBe(1)
```

Comment:
test-library warns against using container this way: https://testing-library.com/docs/react-testing-library/api#container-1. Not exactly sure how to best replace it, but perhaps something like
```suggestion
      const { container } = render(<Form type="SingleField" />)
      await waitFor(() => {
        expect(screen.getAllByRole("input").length).toBe(1)
```